doc/research/06_acting_implementation.md

```diff
@@ -0,0 +1,61 @@
+# Requirements and challenges for an acting implementation
+
+**Summary:** On this page you can find the results of the basic research on acting summed up into resulting requirements and function, that were already agreed upon.
+
+---
+
+## Authors
+
+Gabriel Schwald
+
+### Date
+
+20.11.2022
+
+---
+[[TOC]]
+
+This document sums up all functions (regarding acting) already agreed upon in [#24](https://github.com/ll7/paf22/issues/24), that could be implemented in the next sprint.
```

Comment:
A reference to the research on acting (05_acting.md) would be quite good.

Reply:
Reference added in 9970769